pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -54,7 +56,14 @@ func addDefaultingFuncs(scheme *kruntime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfiguration) {
+func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfiguration, featureGate featuregate.FeatureGate) error {
+	// Make a local copy of the feature gates and combine it with the gates set by this configuration.
+	// This allows us to validate the config against the set of gates it will actually run against.
+	localFeatureGate := featureGate.DeepCopy()
```

Comment:
LOL, interesting insight!  I didn't realize that we are validating the config, so gates are not fully set yet!